transcom/mymove
# B 20757 int

## [B-20757](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A983881)

## Summary

This task updates the error message for DTOD so it is clear why the PPM cannot progress. Follow-on tasks will provide a more granular error message based on reason for failure.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

NOTE: Prior to test I recommend adding "distanceFloat = -1" after line 121 of dto_zip5_distance.go to simulate a DTOD failure.

1. Access the milmove application
2. Login as a Customer
3. Create a PPM
4. Fill in all field for the ppm and progress to the estimated incentives page (use bad zips if you have them, refer to testing note for guidance on how to simulate failures)
5. Fill in estimated weight
6. Save and Continue
7. Verify that error message matches the error message in the acceptance criteria for [B-20757](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A983881)

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
